### PR TITLE
guides for Managed Stacks MySQL cluster

### PR DESCRIPTION
**Pull request by popawu**
guides for Managed Stacks MySQL cluster :

- Create and use MySQL cluster with RunAbove Manager
- Create and use MySQL cluster with API

### CODE REVIEW

**Comment by cyrbil** [comment]
Dates on images links are invalid: 
2015-12-08 should be 2015-12-10.

**Comment by popawu** [comment]
fixed.
thx cyrbil

File: en/_posts/2015-12-10-create-mysql-managed-stack-api.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title:  "How to create a MySQL Managed Stack using the API"
+categories: Labs
```
Comment (VincentCasse) [inline en/_posts/2015-12-10-create-mysql-managed-stack-api.md]: Each lab is a category. Please add a "ManagedStack" category

File: en/_posts/2015-12-10-create-mysql-managed-stack.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "How to create a MySQL Managed Stack"
+categories: Labs
+author: popawu
+lang: en
+---
+
+In this guide you will learn how to subscribe to the Managed Stacks lab and create your MySQL cluster.
```
Comment (VincentCasse) [inline en/_posts/2015-12-10-create-mysql-managed-stack.md]: Can you add a small description to explain some biggest feature of this lab?

File: en/_posts/2015-12-10-create-mysql-managed-stack-api.md (hunk 1 above)
Comment (popawu) [inline en/_posts/2015-12-10-create-mysql-managed-stack-api.md]: done

File: en/_posts/2015-12-10-create-mysql-managed-stack.md (hunk 2 above)
Comment (popawu) [inline en/_posts/2015-12-10-create-mysql-managed-stack.md]: done

**Comment by VincentCasse** [comment]
Ok, that's good for me. Merged

**Pull request merged**